src/api/spec/models/project_spec.rb

```diff
@@ -628,4 +628,95 @@ def reset_project_in_backend
       expect(new_xml).to include('title' => 'Mine', 'description' => {}, 'name' => project.name)
     end
   end
+
+  describe '#categories' do
+    let(:project) { create(:project) }
+    let(:attrib_namespace) { AttribNamespace.create!(name: 'OBS') }
+
+    subject do
+      project.categories
+    end
+
+    context 'when there are quality categories attributes set for the project' do
+      let!(:category_attrib_type) do
```

Comment:
:thought_balloon: Those three `let!`'s should be a `before` block.

Reply:
Done

Reply:
I left two of them outside as they are dependencies for the rest of lets